From 9f3ec798b4db9e2863609394c0e1592672431c03 Mon Sep 17 00:00:00 2001
From: anhuynh <an.huynh.uj@renesas.rvc.com>
Date: Mon, 13 Mar 2017 14:19:12 +0700
Subject: [PATCH 159/909] soc: renesas: rcar_ems_ctrl: Update for code cleanup

commit cd563af7a67e1b33c9574c1454db2d0cc395c416 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

Add parentheses as recommended

Signed-off-by: anhuynh <an.huynh.uj@renesas.rvc.com>
Signed-off-by: Dien Pham <dien.pham.ry@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/soc/renesas/rcar_ems_ctrl.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/soc/renesas/rcar_ems_ctrl.c b/drivers/soc/renesas/rcar_ems_ctrl.c
index bc6ce606cd29..ca9af73da62a 100644
--- a/drivers/soc/renesas/rcar_ems_ctrl.c
+++ b/drivers/soc/renesas/rcar_ems_ctrl.c
@@ -151,8 +151,8 @@ static int __init rcar_ems_ctrl_init(void)
 		return 0;
 	}
 
-	if (ems_poll == 0 ||
-	    ems_mode_on_temp == 0 || ems_mode_off_temp == 0) {
+	if ((ems_poll == 0) ||
+		(ems_mode_on_temp == 0) || (ems_mode_off_temp == 0)) {
 		pr_err("thermal emergency: not set value\n");
 		return 0;
 	}
-- 
2.17.1

